packages/teams-js/eslint-rules/eslint-plugin-recommend-no-namespaces/recommendNoNamespaces.js

```diff
@@ -0,0 +1,28 @@
+module.exports = {
+  meta: {
+    type: 'suggestion',
+    docs: {
+      description: 'Recommend against the usage of namespaces as they are not treeshakable',
+      category: 'Best Practices',
+      recommended: true,
+    },
+    fixable: true,
```

Comment:
Does this rule definition need to have a `fix` section in the `create` part in order to make it fixable? Is this a rule we really want to enable auto-fixing on?